xblock/test/test_fields_api.py

```diff
@@ -206,6 +220,23 @@ def test_mutation_without_save_makes_non_default(self):
         self.mutate(self.get())
         assert_false(self.is_default())
 
+    def test_mutate_pointer_after_save(self):
+        pointer = self.get()
+        self.mutate(pointer)
+        self.block.save()
```

Comment:
Maybe add an `assert_equals` here too? And perhaps a comment on the next line that we're testing what happens if you mutate a pointer without retrieving it through `.get`?